src/Exceptionless.Core/Jobs/EventPostsJob.cs

```diff
@@ -138,51 +168,43 @@ public class EventPostsJob : QueueJobBase<EventPost> {
             }
 
             foreach (var requeueEvent in eventsToRetry) {
+                string contentEncoding = null;
+                byte[] data = requeueEvent.GetBytes(_jsonSerializerSettings);
+                if (data.Length > 1000) {
+                    data = await data.CompressAsync().AnyContext();
+                    contentEncoding = "gzip";
+                }
```

Comment:
@ejsmith I'm wondering if we should even be recompressing it right here.. If it's under our max allowed bytes we probably shouldn't even be dequeueing it.. If we don't compress it here, the check above to get metadata will instantly complete it.